protocols/selenium-monitor/pom.xml

```diff
@@ -11,7 +11,9 @@
   <name>OpenNMS :: Protocols :: Selenium</name>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <seleniumVersion>2.53.1</seleniumVersion>
+      <!-- <seleniumVersion>3.141.59</seleniumVersion> --> <!-- latest version available but doesnt work with htmlunit -->
```

Comment:
Do we need selenium monitor related changes here ? Perhaps just revert 6464c597fdd9bac69a469095cdabd680182af54d ?